components/EpisodeDetails/index.js

```diff
@@ -14,41 +16,79 @@ const StyledArticle = styled.article`
   border-radius: 10px;
 `;
 
+const StyledDetailsTitle = styled.h2`
+  text-align: right;
+  font-size: 2rem;
+`;
+
+const StyledReturnButton = styled.button`
+  position: absolute;
+  top: 0.5rem;
+  left: 0.5rem;
+  border: none;
+  background: none;
+  width: fit-content;
+  height: fit-content;
+`;
+
 const StyledImage = styled(Image)`
-  width: 90%;
+  width: 100%;
   height: auto;
   align-self: center;
+  margin: 1rem;
+`;
+
+const StyledNoContentMessage = styled.p`
+  margin-top: 0.8rem;
+  color: lightgrey;
 `;
 
 export default function EpisodeDetails({ episode }) {
   const router = useRouter();
   const {
-    nummer,
-    titel,
-    autor,
-    hörspielskriptautor,
-    veröffentlichungsdatum,
+    nummer: number,
+    titel: title,
+    autor: author,
+    hörspielskriptautor: scriptauthor,
+    beschreibung: description,
+    veröffentlichungsdatum: releasedate,
+    kapitel: chapters,
     links,
   } = episode;
 
-  const splitDate = veröffentlichungsdatum.split("-");
-  const newDate = splitDate[2] + "." + splitDate[1] + "." + splitDate[0];
+  const splitDate = releasedate.split("-");
+  const formattedReleaseDate =
+    splitDate[2] + "." + splitDate[1] + "." + splitDate[0];
 
   return (
     <StyledArticle>
-      <button type="button" onClick={() => router.back()}>
+      <StyledReturnButton type="button" onClick={() => router.back()}>
```

Comment:
You can shorten this to:
`onClick={router.back}`

Reply:
changed it